### PR TITLE
files

### PR DESCRIPTION
**Pull request by Surya1231**
A Website For Codeshows

### CODE REVIEW

**Comment by rishabhKalakoti** [comment]
Make a different section for team
And personally, I don't want a developer section as it is now open to all for contributions, the contributions will be reflected in Github. But if you want it, you can keep them for now.

**Comment by rishabhKalakoti** [comment]
Also put the css in a different file

File: main.html [hunk 1]
```diff
@@ -0,0 +1,1078 @@
...
+      .developer p {
+        margin-top: 0;
+      }
+
+      .developer img:hover{
+        transform: scale(1.1);
+      }
+
+
+    </style>
```
Comment (rishabhKalakoti) [inline main.html]: Put css in a different file and import it.

File: main.html [hunk 2]
```diff
@@ -0,0 +1,1078 @@
...
+
+                                So come forward and be a part of CodeShows family.</p>
+                                    </div>
+
+                                    <div class="clear"></div>
+                                  </div>
+
+                                  <div class="main">
+                                    <div class="heading">CODESHOWS TEAM</div>
+                                    <hr>
```
Comment (rishabhKalakoti) [inline main.html]: Move teams to a different section

File: main.html [hunk 3]
```diff
@@ -0,0 +1,1078 @@
...
+                                <div class="_2cuy _3dgx _2vxa"><span class="_4yxo">Famous CP websites</span></div>
+                                <pre class="_19ik" dir="ltr"><span class="_2cuy _19ii _2vxa"><a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.codechef.com%2F%3Ffbclid%3DIwAR1d_Lh651dV369_PrvuVbRvhM4aTQa7610wOU1tebdrnd4WOo7GHCSxC10&amp;h=AT1etTOkmjtpK2Q7PYd9f4fqEoeyM541C57maAIovgp9s6koQ1cLY01MfGwO9gk483xBg4ClC3PAiXjsVylBj6Wk21vtcmZwbD5FrsD4HaDe1zpdINfb71PgCXeIWKzy66fTQw" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">codechef</a><br /><a href="https://l.facebook.com/l.php?u=http%3A%2F%2Fcodeforces.com%2F%3Ffbclid%3DIwAR1RsyFmpwuq-A4fbVXf_BzZ4jXH6CZHq7GoJEdvyTSR_a9W7wvV1NguCZY&amp;h=AT2TN8wtZDqYlppn3iwvu2Re0_2zHZoW08VYBHOOTLSKupjD_z_P_YYwov7UWkU8OkmMd2yWzpyafc4dKXrenIoSWFbTKiw_eI1TYGPsEhVFkYPkOqmY9-RIXNl97IqHx0C7kw" target="_blank" rel="noopener nofollow" data-lynx-mode="asynclazy">codeforces</a><br /><a href="https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.hackerrank.com%2Fdashboard%3Ffbclid%3DIwAR3Ji1sNZ4ZpKoORg2_A18zkqEgKMIgYOOSMplBVPjR4LaHxhMIqur-lOlo&amp;h=AT2OzHRgtwt8a6SiB1h_FsR4MtpIa5G5oIxma7KLR9bp1Q8AGWhZ-0qijGDH
```
Comment (rishabhKalakoti) [inline main.html]: Remove whatsapp links as now we have switched to telegram

Comment (rishabhKalakoti) [inline main.html]: Change the telegram link to `https://t.me/codeshows`

**Comment by rishabhKalakoti** [comment]
Add to your PR description,
fixes #3  
fixes #2 
so that it closes the above issues on merge

File: main.html (hunk 3 above)
Comment (Surya1231) [reply main.html]: ok i will make all the changes

**Comment by Surya1231** [comment]
fixes #2 
fixes #3

**Comment by ronakgadia** [comment]
You need to write fixes on the PR request not on comment but on the PR request itself. Just edit the request.

**Pull request merged**